common4j/src/main/com/microsoft/identity/common/java/opentelemetry/CertBasedAuthTelemetryHelper.java

```diff
@@ -35,8 +38,16 @@ public class CertBasedAuthTelemetryHelper implements ICertBasedAuthTelemetryHelp
 
     private final Span mSpan;
 
-    public CertBasedAuthTelemetryHelper() {
-        mSpan = OTelUtility.createSpan(SpanName.CertBasedAuth.name());
+    /**
+     * Creates an instance of CertBasedAuthTelemetryHelper.
+     * @param spanContext current span context.
+     */
+    public CertBasedAuthTelemetryHelper(@Nullable final SpanContext spanContext) {
+        if (spanContext != null) {
+            mSpan = OTelUtility.createSpanFromParent(SpanName.CertBasedAuth.name(), spanContext);
+        } else {
+            mSpan = OTelUtility.createSpan(SpanName.CertBasedAuth.name());
+        }
```

Comment:
Alternatively, you can provide two separate constructors: one that takes a NonNull SpanContext and the other that doesn't even take an argument

Reply:
Addressed here: 51e4f91049ddfd1979ec06cc9c4292590937d561